components/centraldashboard/public/components/registration-page.js

```diff
@@ -87,19 +87,31 @@ export class RegistrationPage extends utilitiesMixin(PolymerElement) {
         );
     }
 
+    async pollProfile(times = 10, delay = 500) {
```

Comment:
Do you think it makes sense to remove the parameters here and just add these values as module constants? Since this isn't a general purpose function and only gets used once, it seems like that simplifies things.

Reply:
Yup, will do!